docs/consume-packages/install-use-packages-dotnet-cli.md

```diff
@@ -37,22 +37,24 @@ This article shows you basic usage for a few of the most common dotnet CLI comma
     dotnet add package Newtonsoft.Json
     ```
 
-3. After the command completes, look at the project file to make sure the package was installed.
+1. After the command completes and you save the project, you can open the project file to see the package reference.
```

Comment:
Given that there's no active process, there's no saving required. It's updated immmediately.